theme/ckeditor5-ui/components/icon/icon.css

```
@@ -29,7 +29,9 @@
 		/* Allows dynamic coloring of the icons. */
 		color: inherit;
 
-		/* Needed by FF. */
-		fill: currentColor;
+		:not([fill]) {
```

It should be 

```suggestion
		&:not([fill]) {
```

for the [nesting plugin](https://www.npmjs.com/package/postcss-nesting) to work.